core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java

```diff
@@ -534,6 +535,15 @@ public static void copyNode(Session session, String srcWorkspaceName, String src
       try {
         workspace.copy(srcPath, destPath);
         Node destNode = (Node) session.getItem(destPath);
+
+        if(destNode.isNodeType(ActivityTypeUtils.EXO_ACTIVITY_INFO)) {
+          destNode.removeMixin(ActivityTypeUtils.EXO_ACTIVITY_INFO);
+        }
+        if(destNode.isNodeType("eoo:onlyofficeFile")) {
```

Comment:
Could you refactor and put those strings in static variables ?

Reply:
I did it